docs/CRSF_SENSORS.md

```diff
@@ -0,0 +1,41 @@
+1   | 1RSS*       | Receiver RSSI                                                           | mLRS        |
+2   | 2RSS        | Not supported                                                           |             |
+3   | Alt         | Altitude                                                                | FC          |
+4   | ANT         | Active receiver antenna                                                 | mLRS        |
+5   | Bat%        | Battery percent remaining                                               | FC          |
+6   | Capa        | Current consumption                                                     | FC          |
+7   | Curr        | Current draw                                                            | FC          |
+8   | FM          | Flight mode                                                             | FC          |
+9   | GPS         | GPS coordinates                                                         | FC          |
+10  | GSpd        | Ground speed                                                            | FC          |
+11  | Hdg         | Heading                                                                 | FC          |
+12  | Ptch        | FC pitch angle                                                          | FC          |
+13  | RFMD        | Receiver update rate (19, 31, 1 - 50Hz)                                 | mLRS        |
+14  | Roll        | FC roll angle                                                           | FC          |
+15  | RQly        | Receiver link quality                                                   | mLRS        |
+16  | RRSP        | Receiver RSSI percentage                                                | mLRS        |
+17  | RSNR        | Not supported                                                           |             |
+18  | RxBt        | Battery Voltage                                                         | FC          |
+19  | Sats        | GPS satellites acquired                                                 | FC          |
+20  | TFPS        | Transmitter update rate (20, 30, 50)                                    | mLRS        |
+21  | TPWR        | Transmitter power (mW)                                                  | mLRS        |
+22  | TPw2**      | Transmitter power (CRSF power level)                                    | mLRS        |
+23  | TQly        | Tramsmitter link quality                                                | mLRS        |
+24  | TRSP        | Transmitter RSSI percentage                                             | mLRS        |
+25  | TRSS*       | Transmitter RSSI                                                        | mLRS        |
+26  | TSNR        | Transmitter signal to noise ratio                                       | mLRS        |
+27  | VSpd        | Vertical speed (barometer)                                              | FC          |
```

Comment:
this is actually not the barometer (there is an extra BARO_ALT crsf message for that specifically)
it's what ardupilot comes up with after all its sensor fusion EKF thingy

Reply:
Will remove the the '(barometer' piece